static/app/views/performance/metricsSearchBar.tsx

```diff
@@ -0,0 +1,108 @@
+import {useEffect, useState} from 'react';
+import {ClassNames} from '@emotion/react';
+import memoize from 'lodash/memoize';
+
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import SmartSearchBar from 'app/components/smartSearchBar';
+import {NEGATION_OPERATOR, SEARCH_WILDCARD} from 'app/constants';
+import {t} from 'app/locale';
+import {Organization, Tag} from 'app/types';
+import useApi from 'app/utils/useApi';
+import {MetricTagValue} from 'sentry/types/metrics';
+
+const SEARCH_SPECIAL_CHARS_REGEXP = new RegExp(
+  `^${NEGATION_OPERATOR}|\\${SEARCH_WILDCARD}`,
+  'g'
+);
+
+type Props = Pick<
+  React.ComponentProps<typeof SmartSearchBar>,
+  'onSearch' | 'onBlur' | 'query' | 'maxQueryLength' | 'searchSource'
+> & {
+  orgSlug: Organization['slug'];
+  projectIds: number[] | readonly number[];
+};
+
+function MetricsSearchBar({
+  orgSlug,
+  onSearch,
+  onBlur,
+  maxQueryLength,
+  searchSource,
+  projectIds,
+}: Props) {
+  const api = useApi();
+  const [tags, setTags] = useState<{key: string}[]>([]);
```

Comment:
Should this type be `MetricTag` instead?

Reply:
good catch :)